architecture/networking/routes.adoc

```diff
@@ -337,13 +337,20 @@ spec:
 <1> The path is the only added attribute for a path-based route.
 ====
 
+[NOTE]
+====
+The service should be able to serve the path specified on the path-based route.
```

Comment:
Not sure why this is needed. The above description lays out the rules with examples. I don't think of a service in this context. The external user provides a fqn/path and, when matched by haproxy, the request ends up on one of the endpoints that the service provided to haproxy for the fqn/path.

Reply:
Agreed, we can get rid of this.